cypress/integration/login.js

```diff
@@ -5,41 +5,25 @@ const loginPage = new LoginPage();
 const timeLoggingPage = new TimeLoggingPage();
 
 describe('Sourcebooks login', function() {
-  it('Should display validation for empty user after attempted loggin', function() {

```

Comment:
please do not delete anything from previous test cases

Reply:
Added missing test case.